var/spack/repos/builtin/packages/tiramisu/package.py

```diff
@@ -0,0 +1,71 @@
+# Copyright 2013-2022 Lawrence Livermore National Security, LLC and other
+# Spack Project Developers. See the top-level COPYRIGHT file for details.
+#
+# SPDX-License-Identifier: (Apache-2.0 OR MIT)
+from spack.package import *
+
+
+class Tiramisu(CMakePackage, CudaPackage, PythonExtension):
+    """Tiramisu is a polyhedral compiler for dense and sparse deep learning
+    and data parallel algorithms.It provides a simple C++ API for expressing
+    algorithms and how these algorithms should be optimized by the compiler."""
+
+    homepage = "http://tiramisu-compiler.org"
+    url = "https://github.com/Tiramisu-Compiler/tiramisu"
+    git = "https://github.com/Tiramisu-Compiler/tiramisu.git"
+
+    maintainers = ["wraith1995"]
```

Comment:
```suggestion
    maintainers("wraith1995")
```
We have a new syntax that appends instead of overwrites, so I won't be removed from the list